### PR TITLE
 Remove `@nguniversal/*` projects

### PR DESCRIPTION
**Pull request by gkalpak**
Previously, the `@nguniversal/*` packages were imported into standard `@angular/cli` apps. However, these packages are not supposed to be used in Angular apps (on the client) and thus do not need to be processed by `ngcc`. They contain code that must be run on the server (with access to Node.js APIs) and thus the corresponding projects were failing during build, due to missing built-in Node.js modules (e.g. `fs`).

We should definitely test how ngUniversal apps work with Ivy (and `ngcc`), but this is not the right way. Removing the packages for now.

_EDIT:
There are tests for ngUniversal+Ivy in the [ngUniversal repo][1] and in the [`@angular/cli` repo][2]. So, we seem to be covered 🌻_

##
This sits on top of #385. Only the last commit is new.

[1]: https://github.com/angular/universal/tree/5798f198d/integration
[2]: https://github.com/angular/angular-cli/blob/8ea892cb7/tests/legacy-cli/e2e/tests/build/platform-server.ts

### CODE REVIEW

File: projects/mydatepicker-ngcc/e2e/protractor.conf.js [hunk 1]
```diff
@@ -13,10 +13,7 @@ exports.config = {
     './src/**/*.e2e-spec.ts'
   ],
   capabilities: {
-    browserName: 'chrome',
-    chromeOptions: {
-      args: [ "--headless", "--disable-gpu"]
-    }
+    'browserName': 'chrome'
```
Comment (alan-agius4) [inline projects/mydatepicker-ngcc/e2e/protractor.conf.js]: Is this intended?

File: .circleci/config.yml [hunk 2]
```diff
@@ -32,11 +32,7 @@ jobs:
       - run:
           name: Install Dependencies
           command: |
-            yarn install --frozen-lockfile
```
Comment (alan-agius4) [inline .circleci/config.yml]: Thanks for this.

File: projects/mydatepicker-ngcc/e2e/protractor.conf.js (hunk 1 above)
Comment (gkalpak) [reply projects/mydatepicker-ngcc/e2e/protractor.conf.js]: These are from #385 (only the last commit is relevant for this PR).

**Comment by mgechev** [comment]
@gkalpak would you rebase?

**Comment by gkalpak** [comment]
@mgechev: Done.

**Comment by gkalpak** [comment]
FYI: Rebased on master.

**Pull request merged**